rest/client.py

```diff
@@ -44,17 +36,43 @@ def order_book(self, currency_pair: str, group: int = 1):
         if group not in [0, 1, 2]:
             raise ValueError('Group parameter should be 0, 1 or 2.')
 
-        rsp = requests.get(url=self.base_endpoint + '/order_book/' + currency_pair, params={'group': group})
-        if rsp.status_code == 404:
-            raise ValueError('Order book for requested currency pair not found.')
+        params = {'group': group}
+        return self._make_request('GET', '/order_book/' + currency_pair, params)
 
-        return rsp.json()
+    def transactions(self, currency_pair: str, period: str = "hour"):
```

Comment:
Again, under pep8 directive, double quotes `"` aren't acceptable for string defintiions. Please use single quotes. Double quotes can only be used for docstrings.